src/index.js

```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
+// import CalculatorPage from './pages/CalculatorPage';
 import reportWebVitals from './reportWebVitals';
```

Comment:
- [x] It would really be great if you can take out all unused code, what do you think? 🤔

Reply:
I checked everything else! Oh well, I'll do that right now. Thanks! 😅